src/main/java/pl/med/clinic/dto/PatientDtoRequest.java

```diff
@@ -0,0 +1,17 @@
+package pl.med.clinic.dto;
+
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@AllArgsConstructor
```

Comment:
sprawdź proszę czy jak dasz AllArgsConstructor to jest potrzebny Setter?? Ogólnie to jest to zależne od biblioteki Jackson, która mapuje obiekty

Reply:
bez setterów save i update działają, więc usuwam